scripts/ci-frontend-metrics.sh

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
+set -eo pipefail
 
 echo -e "Collecting code stats (typescript errors & more)"
 
 ERROR_COUNT_LIMIT=398
```

Comment:
think this needs to be around 580 or something (as we changed the error count command from --strictNullChecks to just --strict)

Reply:
@torkelo OK, will make the change.

Reply:
Took care of it.